cmake/ExternalProjects.cmake

```diff
@@ -111,7 +111,7 @@ FetchContent_Declare(wavm_ext
 
 FetchContent_Declare(wamr_ext
     GIT_REPOSITORY "https://github.com/faasm/wasm-micro-runtime"
-    GIT_TAG "135672cab24d877db4ef6933c0ab150351384d51"
+    GIT_TAG "a31e5a4fa299c4f8384f40e157b0a928ad0bda1b"
```

Comment:
See [this comment](https://github.com/faasm/faasm/pull/728/files#r1125522410) (if you have not seen it yet).